requirements.txt

```diff
@@ -1,7 +1,7 @@
 googlemaps==2.5.1
 nltk==3.4.5
 pandas==0.25.1
-psycopg2==2.8.5
+psycopg2-binary==2.8.5
```

Comment:
Building Psycopg requires a few prerequisites (a C compiler, some development packages):
> ![image](https://user-images.githubusercontent.com/39518771/82837532-f814c980-9ee6-11ea-8536-58dafa7d1978.png)

The alternative as [suggested:](https://pypi.org/project/psycopg2/)
> ![image](https://user-images.githubusercontent.com/39518771/82837811-d2d48b00-9ee7-11ea-8a0f-3b8bfc0ec1b5.png)